credentialdigger/client_postgres.py

```diff
@@ -195,6 +198,48 @@ def add_rule(self, regex, category, description=''):
             query='INSERT INTO rules (regex, category, description) \
                     VALUES (%s, %s, %s) RETURNING id')
 
+    def add_embedding(self, discovery_id, embedding=None, repo_url=''):
```

Comment:
Can an embedding have an empty repo url? Isn't it an external key in the db?

Reply:
okay, I'll remove the default value